lib/Model/EventSchedule.php

```diff
@@ -10,16 +10,20 @@
 use InvalidArgumentException;
 use function sprintf;
 
+/**
+ * @template-extends AbstractLazyCollection<int, \Doctrine\Website\Model\EventScheduleSlot>
+ */
 final class EventSchedule extends AbstractLazyCollection
 {
     /** @var mixed[] */
     private $event;
 
-    /** @var EventSpeakers */
+    /** @var EventSpeakers<EventSpeaker> */
```

Comment:
Does this really have to be that way? Phpstan should be able to figure that `EventSpeakers` elements can't be anything but `EventSpeaker` instances, shouldn't it?

Reply:
@greg0ire Done. It became obsolete because of `@template-extends` on the class.